clubtech.py/nolluda.py

```diff
@@ -0,0 +1,70 @@
+import os
+import time
+
+class Dog:
+    
+    def __init__(self, name, breed, age):
+        self.name = name
+        self.breed = breed
+        self.age = age
+    
+    def bark(self):
```

Comment:
Change all the print statements inside the 5 methods. Preferably, the comma should be before and after the age only